README.md

```diff
@@ -128,8 +129,7 @@ func main() {
     s := server.NewMCPServer(
         "Calculator Demo",
         "1.0.0",
-        server.WithResourceCapabilities(true, true),
-        server.WithLogging(),
```

Comment:
AFAIK this server does not emit logging notifications: https://modelcontextprotocol.io/specification/2025-03-26/server/utilities/logging#capabilities